.github/workflows/build-cli.yml

```diff
@@ -138,3 +138,38 @@ jobs:
           name: bws-macos-universal-${{ env._PACKAGE_VERSION }}.zip
           path: ./bws-macos-universal-${{ env._PACKAGE_VERSION }}.zip
           if-no-files-found: error
+
+  third_party:
+    name: Generate THIRDPARTY.html
+    runs-on: ubuntu-22.04
+    needs:
+      - setup
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Install rust
+        uses: dtolnay/rust-toolchain@0e66bd3e6b38ec0ad5312288c83e47c143e6b09e # stable
```

Comment:
❓ Are all the new Actions used here "vetted" in the sense of popularity, safety / security, etc.?

⚠️ Renovate uses tags for the version comments and I don't see this one so unsure this will work as expected.